uwsgi-master.pid

```diff
@@ -0,0 +1 @@
+15991
```

Comment:
Add this file to `.gitignore` and remove this file from this PR.

Reply:
ok